File: ReadIt/ViewModels/CommentModel.cs
namespace ReadIt.ViewModels
{
    public class CommentModel
    {
        public long Id { get; set; }

        public string Text { get; set; }

        public long BlogId { get; set; }

        public long? CreatedBy { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Website { get; set; }
        public UserModel User { get; set; }
        public DateTime CreatedOn { get; set; }
    }
}

File: ReadIt.Angular/src/app/core/models/comment.model.ts
import { UserModel } from "./user.model"

export interface CommentModel {
    id: number
    text: string
    blogId: number
    createdBy: number | null
    user?: UserModel | null
    name?:string | null
    email?:string | null
    website?: string | null
    createdOn: Date
}
